named-tunnel-k8s/README.md

```diff
@@ -1,66 +1,63 @@
-This is an example of using a [named tunnel (beta)](https://community.cloudflare.com/t/argo-tunnel-named-tunnel-beta/202445) in kubernetes. 
+This is an example of using a [named tunnel (beta)](https://community.cloudflare.com/t/argo-tunnel-named-tunnel-beta/202445) in Kubernetes.
```

Comment:
Why does it say `(beta)`?

Reply:
The guide was originally written when Named Tunnels were in beta -- removed.